src/main/java/io/github/christophermanahan/carnitas/Main.java

```diff
@@ -4,23 +4,25 @@
 import java.net.ServerSocket;
 
 public class Main {
-    private static Acceptor connectionAcceptor;
+    private static Listener connectionListener;
 
     public static void main(String[] args) {
         int port = args.length == 0 ? 33333 : Integer.parseInt(args[0]);
```

Comment:
It feels like this should be wrapped in a function to make it more descriptive as to what's going on

Reply:
I have a story that will obviate the need for this field that I will be working on today!